src/detailPage/DetailPage.js

```diff
@@ -62,7 +59,7 @@ export default function DetailPage(allParametersObj) {
         />
       </Container>
       <StyledNavCloseButton onClick={() => onClick(history)}>
-        <StyledCloseIcon src={CloseIcon} />
+        <Close name={'close'} size={30} />
```

Comment:
Maybe "CloseDetailPage" or "CloseButton" or something similar makes it a bit more clear, what this Component is for.